README.md

```diff
@@ -24,14 +28,15 @@ end
 ```
 
 PLEASE NOTE: Without performing one of the above license acceptance steps, all other resources in the habitat cookbook will fail with an error prompting that the license must be accepted.
+
 ## Requirements
 
 ### Platforms
 
 - RHEL 6+
-- Ubuntu 14.04+
-- Debian 7+
-- Windows 2016
+- Ubuntu 16.04+
+- Debian 9+
+- Windows 2016+
 
 ### Habitat
 
```

Comment:
The line under this has `- Habitat version: 0.88.0` is this correct?

Reply:
It is not. We are currently pinned to `1.5.0`. I plan to update that to current soon but, wanted to have a discussion about allowing it to always install current unless otherwise specified. 
@jonlives @tas50 I'd like your thoughts on that. I think we should be pushing latest and leave it on the customer to decide what version they want to pin to. Do we have some caveats to that?